app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java

```diff
@@ -87,6 +92,21 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         }
     }
 
+    @Override
+    protected Adapter onCreateAdapter(PreferenceScreen preferenceScreen) {
+        return new PreferenceGroupAdapter(preferenceScreen) {
+            @Override
+            public void onBindViewHolder(PreferenceViewHolder holder, int position) {
+                super.onBindViewHolder(holder, position);
+                Preference preference = getItem(position);
+                View iconFrame = holder.itemView.findViewById(R.id.icon_frame);
+                if (iconFrame != null) {
+                    iconFrame.setVisibility(View.GONE);
+                }
```

Comment:
@SaumiaSinghal Can you use Butterknife here?

Reply:
Yes sure @maskaravivek

Reply:
I am unable to use Butterknife here. Using `Butterknife.bind(holder.itemview)` does not seem to work.

Reply:
That would need to be done in the PreferenceViewHolder's constructor

Reply:
Right, but I cannot edit PreferenceViewHolder class

Reply:
Yes, seems like that not an option

Reply:
Seems like your reviews are drawn back, is this the case @ashishkumar468 @maskaravivek ? Are you good with these changes now?

Reply:
> @maskaravivek and @ashishkumar468 can you please re-consider your reviews?

Pinging @ashishkumar468 :)